src/System.Memory/tests/Performance/Perf.Base64EncodeDecode.cs

```diff
@@ -128,6 +128,7 @@ private static void Base64DecodeDetinationTooSmall(int numberOfBytes)
             }
         }
 
+#if netcoreapp
```

Comment:
If there is no longer a netcoreapp configuration, did you make this code dead? And similar places.

Reply:
thanks, that's a mistake from my side.